react/week4/managing-state/src/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import * as serviceWorker from './serviceWorker';
+
+// Importing components
+import Context from './context';
+import Searchuser from './searchuser';
+import ListItems from './userListItems';
+
+// Context
+const Contextprovider = Context.Provider;
+
+function FetchArray(event){
+  const URL=`https://api.github.com/search/users?q=${event}`;
+    return fetch(URL).then(response => response.json())        
+}
+
+class TodoList extends React.Component{
```

Comment:
You're not using this component to store todos, it's generating a list of users from github. I would rename this component and all uses of the word todo to correctly reflect what you're actually rendering in this component.

Reply:
I used few things from my old homework that's why the name was "Todo", but i have changed it in every file. Will keep it in mind for future